six/modules/python/six.sicd/source/six_sicd.i

```diff
@@ -306,6 +308,19 @@ def readRegion(inputPathname, startRow, numRows, startCol, numCols, schemaPaths
     getWidebandRegion(inputPathname, schemaPaths, complexData, startRow, numRows, startCol, numCols, widebandBuffer)
 
     return widebandData, complexData
+
+def writeXML(complexData, pathname):
```

Comment:
Typically I'd just think of Swig code going in the .i file but I guess we don't have a good precedent for utility functions like this going elsewhere?  Would it be straightforward to put this in its own .py file somewhere that waf would still install?  If that's a big pain, it's ok to leave it here for now.